### DISCUSSION TITLE
Questions regarding variables

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1562] [Question and answer]
I wonder where the values for these variables are: btc_type_sleepingbag, btc_type_seat, btc_type_satelliteAntenna, btc_type_power,  btc_type_tent.

Cheers.

**Comment by Vdauphin** [comment]
Hello,
https://github.com/search?q=repo%3AVdauphin%2FHeartsAndMinds+%22btc_type_sleepingbag+%3D%22&type=code
Cheers
